assets/app/views/browse/_deployment-details.html

```diff
@@ -52,47 +52,12 @@
   </dd>
   <dt>Replicas:</dt>
   <dd>
-    <div ng-show="!replicas.editing">
-      {{deployment.status.replicas}} current / {{deployment.spec.replicas}} desired
-      <a href="" title="Edit" class="action-button" ng-click="replicas.desired = deployment.spec.replicas; replicas.editing = true">
-        <i class="icon icon-pencil" style="margin-left: 5px;"></i>
-        <span class="sr-only">Edit</span>
-      </a>
-    </div>
-    <div ng-show="replicas.editing">
-      <form name="replicas.form" ng-submit="scale()" class="replicas-form">
-        <span ng-class="{'has-error': replicas.form.$invalid}">
-          <input type="number"
-                 name="desired"
-                 ng-model="replicas.desired"
-                 ng-required="true"
-                 min="0"
-                 ng-pattern="/^\-?\d+$/"
-                 focus-when="{{replicas.editing}}"
-                 select-on-focus
-                 class="form-control input-number">
-        </span>
-        <a href="" title="Scale" class="action-button" ng-click="scale()">
-          <i class="icon icon-ok action-button" style="margin-left: 5px;"></i>
-          <span class="sr-only">Scale</span>
-        </a>
-        <a href="" title="Cancel" class="action-button" ng-click="cancelScale()">
-          <i class="icon icon-remove action-button" style="margin-left: 5px;"></i>
-          <span class="sr-only">Cancel</span>
-        </a>
-        <div ng-if="replicas.form.$invalid" class="has-error">
-          <div ng-if="replicas.form.desired.$error.required" class="help-block">
-            A value for replicas is required.
-          </div>
-          <div ng-if="replicas.form.desired.$error.pattern" class="help-block">
-            Replicas must be a whole number.
-          </div>
-          <div ng-if="replicas.form.desired.$error.min" class="help-block">
-            Replicas can't be negative.
-          </div>
-        </div>
-      </form>
-    </div>
+    <!-- Enable scaling if this is a plain replication controller or it's the active deployment. -->
+    <replicas status="deployment.status.replicas"
+              spec="deployment.spec.replicas"
+              disable-scaling="(deployment | isDeployment) && (!isActive || !deploymentConfig)"
```

Comment:
if the deploymentConfig is deleted we should let you scale so you can can scale to zero to delete the deployment